.mailmap

```diff
@@ -565,6 +565,10 @@ Robert Habermeier <rphmeier@gmail.com>
 Robert Millar <robert.millar@cantab.net>
 Roc Yu <rocyu@protonmail.com>
 Rohit Joshi <rohitjoshi@users.noreply.github.com> Rohit Joshi <rohit.joshi@capitalone.com>
+Ross Smyth <18294397+RossSmyth@users.noreply.github.com>
+Ross Smyth <18294397+RossSmyth@users.noreply.github.com> <crs2017@gmail.com>
+Ross Smyth <18294397+RossSmyth@users.noreply.github.com> <rsmyth@electrocraft.com>
+Ross Smyth <18294397+RossSmyth@users.noreply.github.com> Christopher Smyth <crs2017@gmail.com>
```

Comment:
This line shouldn't be necessary, it should be covered by line 568-569 already

Reply:
in detail:
- line 568 ensures that for every commit with the github email, the name is "Ros Smyth".
- line 569 ensure that for every commit with the gmail email, the name is "Ross Smyth" and the email is the github email
- line 570 ensures that for every commit with the electrocraft email, the name is "Ross Smyth" and the email is the github email
- line 570 ensures that for every commit with the gmail email *and* the other name, the name is "Ross Smyth" and the email is the github email

so the last one is redundant

Reply:
aaaaahhhhhh I wish this was written down somewhere because a few minutes of searching did not show anything

Reply:
was the git mailmap documentation not helpful enough? I've been thinking about writing a mailmap guide myself but I'm not sure what I'd put there that the mailmap docs don't explain already.

also I have a suspicion that the thanks mailmap implementation is not correct